public/components/application_analytics/components/app_table.tsx

```diff
@@ -248,59 +213,69 @@ export function AppTable(props: AppTableProps) {
       sortable: true,
       render: (value) => <EuiText>{moment(value).format(UI_DATE_FORMAT)}</EuiText>,
     },
-  ] as Array<EuiTableFieldDataColumnType<ApplicationType>>;
+    {
+      name: 'Actions',
+      actions,
+    },
+  ];
+
+  const filteredApplications = applications.filter((app) =>
+    app.name.toLowerCase().includes(searchQuery.toLowerCase())
+  );
 
   return (
-    <div style={pageStyles}>
+    <>
+      {!newNavigation && (
+        <div style={{ paddingLeft: '16px', paddingTop: '8px' }}>
```

Comment:
Can may be remove these custom styles.

Reply:
Using EUI page to add padding adds additional white space, Have not found a good alternative.
<img width="573" alt="Screenshot 2024-08-22 at 1 23 36 PM" src="https://github.com/user-attachments/assets/2b5fcba9-5e50-43c0-b1ff-c34c56c0b9ca">